### ISSUE TITLE
Who is this _really_ for?

### ISSUE THREAD

**Issue by nickmalcolm** [ICTUNZ/ICTUNZ#1]
Do you actually mean just "developers"? What about developers who start doing management stuff, or are responsible for employing other developers, do they get kicked out? What about if their hat changes to be a sysadmin, or a CTO, or a penetration tester? Does their job title have to be "developer"?

The name of the union, "ICT", implies a broader scope than just "developers", but that isn't reflected in the eligibility section.

**Comment by veb** [comment]
I agree totally with you Nick.

This needs to be better aimed and directed at either just developers, or in
fact the entire IT industry. I think this has a LOT of potential, so I'm
watching. :)

Mike

On Thu, Apr 14, 2016 at 1:14 PM, Nick Malcolm notifications@github.com
wrote:

> Do you actually mean just "developers"? What about developers who start
> doing management stuff, or are responsible for employing other developers,
> do they get kicked out? What about if their hat changes to be a sysadmin,
> or a CTO, or a penetration tester? Does their job title have to be
> "developer"?
> 
> The name of the union, "ICT", implies a broader scope than just
> "developers", but that isn't reflected in the eligibility section.
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub
> https://github.com/ICTUNZ/ICTUNZ/issues/1

## 

Mike Mackenzie, _BInfoTech_

http://veb.co.nz
Dunedin, New Zealand

**Comment by ICTUNZ** [comment]
hmm... I'd like the union to include sysadmins and penetration testers, but developers-who-also-manage and CTOs, etc are a grey area.

**Comment by faceleg** [comment]
The "who is this for" question needs to be answered definitively and documented clearly in the readme

**Comment by nickmalcolm** [comment]
> developers-who-also-manage and CTOs, etc are a grey area.

in which case you'll need to have clear "you must leave the union if your role falls outside of X, Y, Z".

I'd also argue that you'd then lose a lot of experience / insight, but it could be a tradeoff you're willing to make so that the union stays focussed.

**Comment by ICTUNZ** [comment]
Eligibility questions would be decided case-by-case by a membership committee, so it's not necessary to anticipate every possible contingency now, but just to set some guidelines for the membership committee to interpret.

It's my own view that people responsible for hiring/firing developers should not be eligible, but I want to see what other developers think about this before presenting the constitution for a vote.

We do also need to replace "developer" with a word that includes anyone who does technical work... suggestions?

**Comment by veb** [comment]
Yes, replacing developer with a proper term is better. Is a developer a programmer/analyst? Is a developer a software engineer? A software architect? I'm not sure what a good replacement would be.

If your job role is that you develop software yourself, then you are eligible. If you are not, i.e. you are managing developers who develop software you are not directly developing software so you're negligible I guess.

**Comment by nickmalcolm** [comment]
Yeah I have a feeling I'm gonna be outside the scope of your membership based on your responses, so I'll leave it up to y'all to decide. All the best!

**Comment by ICTUNZ** [comment]
@nickmalcolm: take care :-)

**Comment by ICTUNZ** [comment]
@veb: what about developers who are also responsible for hiring/firing other developers... should they be eligible?

**Comment by faceleg** [comment]
At one of my previous roles I was both a programmer and responsible for hiring other programmers. Where would that have placed me?

**Comment by faceleg** [comment]
@veb @ICTUNZ this describes one of my previous roles exactly.

**Comment by veb** [comment]
Hey Michael,

My issue is: a guy who knows HTML may think he's qualified to hire devs.

Then things go wrong. Developer has issues with manager because manager
doesn't understand and so on... What happens here? Union can't go between
members, right?

That's what I mean. :-)

Mike

On Mon, Apr 18, 2016 at 3:54 PM, Michael Robinson notifications@github.com
wrote:

> @veb https://github.com/veb @ICTUNZ https://github.com/ICTUNZ this
> describes one of my previous roles exactly.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/ICTUNZ/ICTUNZ/issues/1#issuecomment-211182328

## 

Mike Mackenzie, _BInfoTech_

http://veb.co.nz
Dunedin, New Zealand

**Comment by faceleg** [comment]
Yeah. My opinion is:
- **Eligible for those who write code as the majority of their role.**
- **Those who perform any aspect of management are to be excluded.**

Being as clear and draconian as this is the only way to remove confusion, which would kill a fledgling organisation like this before it started.

**Comment by veb** [comment]
Agreed. I feel like a dick before about management, but yeah -- there's
definitely great managers who _actually_ code but it's a very tricky one to
figure out IMO.

On Mon, Apr 18, 2016 at 4:05 PM, Michael Robinson notifications@github.com
wrote:

> Yeah. My opinion is:
> 
> Eligible for those who write code as the majority of their role.
> 
> Those who perform any aspect of management are to be excluded.
> 
> Being as clear and draconian as this is the only way to remove confusion,
> which would kill a fledgling organisation like this before it started.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> https://github.com/ICTUNZ/ICTUNZ/issues/1#issuecomment-211189672

## 

Mike Mackenzie, _BInfoTech_

http://veb.co.nz
Dunedin, New Zealand

**Comment by ICTUNZ** [comment]
ok, I've made a few changes... comments?
